source/query/atlas-search.txt

```diff
@@ -25,32 +37,28 @@ Steps
    :style: normal
 
    .. step:: Find the index you want to use in your query
-      
-      a. From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`.
-      #. Hover over the index you want to use in your query.
 
-   .. step:: Run the query
+      From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`. 
+      Hover over the search index you want to use in your query. 
       
-      a. Click the :guilabel:`Aggregate` button.
-
-         |compass| switches to the :guilabel:`Aggregations` tab where you 
-         can run your query. 
-        
-      #. Construct your query.
+   .. step:: Open the aggregation pipeline builder
+      
+      Click the :guilabel:`Aggregate` button that appears when you hover over 
+      your search index.
          
-         By default, |compass| selects the :pipeline:`$search` stage for
-         your query. You can select the  :pipeline:`$searchMeta` stage
-         to use in your query. If you are using a ``vectorSearch`` type
-         index in your query, you can only use the
-         :pipeline:`$vectorSearch` stage in your query.  
+   .. step:: Construct your query
+      
+      If you have an Atlas Search index, add the ``$search`` or 
+      ``$searchMeta`` stage to your aggregation pipeline. If you have an 
+      Atlas Vector Search index, add the ``$vectorSearch`` stage.
 
-         .. note:: 
+      .. note:: 
 
-            The Atlas Search :pipeline:`$search` and :pipeline:`$searchMeta` and 
-            the Atlas Vector Search :pipeline:`$vectorSearch` pipeline stages must 
-            be the first stage in your aggregation pipeline.
+         The Atlas Search ``$search`` and ``$searchMeta`` and 
```

Comment:
(question)
Is it EITHER the Atlas Vector Search pipeline stages, OR the Atlas Search pipeline stages? I'm not sure this makes sense as written, just yet.

Reply:
Ah let me try rewording this note altogether.